chemie/events/migrations/0008_auto_20210215_1657.py

```diff
@@ -0,0 +1,15 @@
+# Generated by Django 2.2.10 on 2021-02-15 15:57
+
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('events', '0007_social_check_in'),
+    ]
+
+    operations = [
+    ]
```

Comment:
Now there are two migrations here both depending on `0007_social_check_in`, so I'm not able to `migrate`. This must be fixed before merging

Reply:
Deleted one of them, but im not sure they are supposed to be here at all.

Reply:
The correct migration (built based on all migrations in master) is now the one commited!